### PR TITLE
Cover a bunch of edge cases with ticks for uniform value graphs

### PR DESCRIPTION
**Pull request by brettimus**
This is an alternative solution for improving y-ticks on graphs of uniform values

Uniform error rate of 0%:

image

Uniform latency of .00475s

image

Some 100% success rates:

image

### CODE REVIEW

File: fiberplane-charts/src/Mondrian/generator/utils/getYAxisForConstantValue.ts [hunk 1]
```diff
@@ -26,6 +40,47 @@ export function getYAxisForConstantValue(value: number): Axis {
...
+  axis.tickSuggestions = [];
+
+  let currentTick = firstTick;
+
+  while (currentTick <= axis.maxValue) {
+    axis.tickSuggestions.push(currentTick);
+    currentTick += interval;
+  }
+
+  return;
```
Comment (arendjr) [inline fiberplane-charts/src/Mondrian/generator/utils/getYAxisForConstantValue.ts]: This `return` seems a bit superfluous :)

**Review by arendjr — APPROVED** [review]
Very nice! :+1:

**Comment by arendjr** [comment]
Btw, I see the PR is against my PR, so if you merge into that one I can update the Rust version if you like.

**Comment by brettimus** [comment]
> Btw

yeah! i'll tidy this up then merge into yours

**Pull request merged**